test/unit/ptable_test.rb

```diff
@@ -49,6 +52,7 @@ class PtableTest < ActiveSupport::TestCase
 
         pt = Ptable.find_by_name(name)
         assert_equal @os_deb.family, pt.os_family
+        assert_equal @new_org.id, pt.organization_ids
```

Comment:
Oh, and this should probably be in a new test - or at least reword the title to mention orgs.